usr/src/cmd/sgs/libld/common/args.c

```diff
@@ -1577,6 +1578,16 @@ parseopt_pass1(Ofl_desc *ofl, int argc, char **argv, int *usage)
 					    MSG_ORIG(MSG_ARG_Z), optarg);
 					return (S_ERROR);
 				}
+			} else if (strncmp(optarg, MSG_ORIG(MSG_ARG_SYSROOT),
+			    MSG_ARG_SYSROOT_SIZE) == 0) {
+				/*
+				 * NB: We don't check this exists, or is a
```

Comment:
It's weird to me too, I had started checking and canonicalizing the path here when I discovered that GNU do not.
Perhaps we should issue a warning, in fact, rather than guidance?